modules/autoattach.cpp

```diff
@@ -116,22 +118,36 @@ class CChanAttach : public CModule {
         }
     }
 
+    void HandleSwap(const CString& sLine) {
+        u_int iA = sLine.Token(1).ToUInt();
```

Comment:
Where does this typedef come from?

Reply:
At least on my machine it is part of the system headers. I was looking at how one of the other modules handled numeric input and copied it from there.